src/draw/lv_draw_image.h

```diff
@@ -26,17 +26,7 @@ extern "C" {
  *      MACROS
  **********************/
 
-/**********************
- *      TYPEDEFS
- **********************/
-
-typedef struct {
-    lv_color_t alpha_color;
-    const lv_color32_t * palette;
-    uint32_t palette_size   : 9;
-} lv_draw_image_sup_t;
-
-typedef struct _lv_draw_image_dsc_t {
+typedef struct lv_draw_image_dsc_t {
```

Comment:
not sure why the struct name and the typedef name are the same or what purpose it serves.